mail/rspamd/Portfile

```diff
@@ -1,79 +1,101 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; truncate-lines: t; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           cmake 1.0
+PortGroup           cmake 1.1
 PortGroup           github 1.0
 
-github.setup        vstakhov rspamd 0.8.3
+github.setup        rspamd rspamd 1.9.2
 
-version             0.8.3
-revision            3
+name                rspamd
+version             1.9.2
+revision            0
 categories          mail
 license             BSD
 maintainers         nomaintainer
 platforms           darwin
 description         Rspamd filtering system is created as a replacement of popular spamassassin \
-                    spamd and is designed to be fast, modular and easily extendable system.
+    spamd and is designed to be fast, modular and easily extendable system.
 long_description    ${description}  Rspamd core is written in C language using event driven \
-                    paradigma. Plugins for rspamd can be written in lua. Rspamd is designed to \
-                    process connections completely asynchronous and do not block anywhere in code.
+    paradigma. Plugins for rspamd can be written in lua. Rspamd is designed to \
+    process connections completely asynchronous and do not block anywhere in code.
 homepage            http://rspamd.com
 
-if {${github.version} eq ${version}} {
-    master_sites        http://rspamd.com/downloads
-    use_xz              yes
-    checksums           rmd160  dc43a24c9eef555bd390ab31c4aa1deb8c5e7704 \
-                        sha256  20f788b08406fdaa8148d726e7ba18940108afff2d48b4f47726e3f2a6a88daf
-} else {
-    fetch.type          git
-    post-fetch {
-        system -W ${worksrcpath} "git submodule update --init"
-    }
-}
+platforms           darwin
+supported_archs     noarch
+license             Apache-2
 
-depends_build-append port:pkgconfig
+distname            ${name}-${version}
```

Comment:
`distname` is already correctly set by the GitHub portgroup. So please delete this line.

Reply:
This is still unresolved.

Reply:
I've fixed this...